R/dvhydrograph-render.R

```
@@ -211,36 +303,67 @@ getDVHydrographPlotConfig <- function(reportObject, info = NULL, ...){
   return(styles)
 }
 
-getDVHydrographRefPlotConfig <- function(reportObject, info = NULL, ...){
+#' Get DV Hydrograph Reference Plot Config
+#'
+#' @description Given an item to plot, fetch the associated
+#' plot feature(s) and their styles. 
+#' @param plotItem the item to fetch styles and plot features for
+#' @param ... any additional parameters to pass into the function
+getDVHydrographRefPlotConfig <- function(plotItem, ...){
   styles <- getDvHydrographStyles()
 
-  x <- reportObject[[1]]$time
-  y <- reportObject[[1]]$value
-  legend.name <- reportObject[[1]]$legend.name
+  x <- plotItem[[1]]$time
+  y <- plotItem[[1]]$value
+  legend.name <- plotItem[[1]]$legend.name
   args <- list(...)
 
-  styles <- switch(names(reportObject), 
-    secondaryRefTimeseries = list(
-      lines = append(list(x=x, y=y, legend.name=legend.name), sref_lines)
+  styles <- switch(names(plotItem), 
```

Same here, let's throw an exception for not found.